x-pack/plugins/profiling/server/lib/setup/security_role.ts

```diff
@@ -28,10 +30,13 @@ export async function setSecurityRole({ client }: ProfilingSetupOptions) {
     name: PROFILING_READER_ROLE_NAME,
     indices: [
       {
-        names: ['profiling-*'],
+        names: ['profiling-*', '.profiling-*'],
         privileges: ['read', 'view_index_metadata'],
       },
     ],
     cluster: ['monitor'],
+    metadata: {
+      version: METADATA_VERSION,
```

Comment:
One suggestion (actually an improvement): If we want this role to show up as a reserved role in the UI, we could add:

```
"_reserved": true
```

in `metadata`.

Reply:
Hmm, too bad. For reference: I had inferred this from looking at the role definition of `kibana_user`.